AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/ModelConstants.java

```diff
@@ -70,4 +71,21 @@ public static String classSuffix() {
 	public static String generationSuffix() {
 		return generationSuffix;
 	}
+
+	public static List<Class<? extends Annotation>> validEnhancedViewSupportAnnotations() {
+		return validEnhancedViewSupportAnnotations;
```

Comment:
shouldn't we return a new or immutable copy of our internal list? just returning the list allows plugins to fully modify the list.

@WonderCsabo wdyt?

Reply:
We should. However they still can modify it through the setters...

Reply:
There are no setters. only "adders". and ADDING stuff is ok, but with the original list you could also remove stuff.

Reply:
True.

Reply:
But also if you adds invalid stuff, that is not a good thing. :)